_posts/2021-05-22-report-arms-spring2021.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title:  "Report: ARMS - reducing robot simulation code by 92%"
+author: "Sean O'Connor"
+categories: [ "robotics", "simulation", "gazebo", "sdf", "javascript"  ]
```

Comment:
These should be "tags" not "categories". See: https://raw.githubusercontent.com/compusciencing/compusciencing.github.io/master/post-template.md